docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -30,6 +30,15 @@ When upgrading to a new major version of Node.js, the following extra steps must
     You can change which Node.js major version to view, by changing the selected branch.
     If Node.js has dropped support for platform still supported by Kibana, appropriate steps must be taken as soon as possible to deprecate support for this platform. This way support for it can be dropped before the currently used major version of Node.js https://github.com/nodejs/release#release-schedule[reaches End-of-Life].
 
+[[custom-nodejs-builds]]
+=== Custom builds of Node.js
+
+Due to Node.js 16 coming to an https://nodejs.org/en/blog/announcements/nodejs16-eol[early End-of-Life] and Node.js 18 no longer supporting the same platforms that {kib} supports (most notably CentOS7/RHEL7), we cannot bundle the official Node.js binaries with the Linux distributable of {kib}.
+To keep support for these older platforms, we're bundling the Linux distributable of {kib} with a https://github.com/elastic/kibana-custom-nodejs-builds[custom build of Node.js] with extended backwards compatibility.
+The only difference between the offical Node.js build and our custom build, is the version of `glibc` that it's compiled against.
+
+For details, see https://github.com/elastic/kibana-custom-nodejs-builds.
```

Comment:
I just realised that there's already a link to this repo above (my bad), so feel free to remove this redundant line

```suggestion
```